Userland/Services/LookupServer/DNSPacket.cpp

```diff
@@ -40,13 +40,14 @@ ByteBuffer DNSPacket::to_byte_buffer() const
         header.set_is_query();
     else
         header.set_is_response();
+    header.set_authoritative_answer(m_authoritative_answer);
     // FIXME: What should this be?
     header.set_opcode(0);
     header.set_response_code(m_code);
     header.set_truncated(false); // hopefully...
-    header.set_recursion_desired(true);
+    header.set_recursion_desired(false);
```

Comment:
No!

We definitely want recursion for regular DNS. You'll have to differentiate between DNS and mDNS here.

Reply:
Yup, that's on me. Should be fixed now.

![Screenshot from 2021-05-08 00-35-48](https://user-images.githubusercontent.com/388571/117514956-5ec6b200-af95-11eb-9ef7-d5b52bc85ccb.png)